doc/user/content/concepts/_index.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Concepts"
-description: "Core concepts in Materialize"
+description: "Learn about the core concepts in Materialize."
```

Comment:
Just little changes to descriptions and aliases.  These metadata descriptions will get better, but agree that even at this stage of the docs, we don't want them to just mimic the title.